lib/factory_girl.rb

```diff
@@ -55,8 +57,16 @@ def self.reset_configuration
     @configuration = nil
   end
 
-  def self.lint(factories_to_lint = FactoryGirl.factories)
-    Linter.lint!(factories_to_lint)
+  def self.lint(factories = nil, options = {})
+    if factories.is_a?(Hash)
+      factories_to_lint = FactoryGirl.factories
+      options = factories
+    else
+      factories_to_lint = factories || FactoryGirl.factories
```

Comment:
does changing the method signature to have this be the default (versus `nil`) allow this just to be `factories`?